__tests__/app.test.js

```diff
@@ -26,3 +27,23 @@ describe("GET /api", () => {
       });
   });
 });
+describe("GET /api/topics", () => {
+  test("200: Responds with an object of all the topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({body}) => {
+        expect(body.result).toEqual(data.topicData)
+      })
+  })
+})
+describe("Not found error", () =>{
```

Comment:
The describe description and test description do not give enough information about what this test is checking.